common/src/primitives.rs

```diff
@@ -963,7 +963,7 @@ mod tests {
     }
 }
 
-#[derive(Copy, Clone)]
+#[derive(Copy, Clone, Debug)]
```

Comment:
I think it's better to derive Debug only when std is on
```suggestion
#[derive(Copy, Clone,)]
#[cfg_attr(feature = "std", derive(Debug))]
```

Reply:
Decided to leave it as is